react-components/src/utilities/FdmSDK.ts

```diff
@@ -8,7 +8,7 @@ import { type FdmPropertyType } from '../components/Reveal3DResources/types';
 type InstanceType = 'node' | 'edge';
 type EdgeDirection = 'source' | 'destination';
 
-type InstanceFilter = any;
+export type InstanceFilter = any;
```

Comment:
Is this used anywhere? Should it be used in a parameter in `useSearchMappedEquipmentFDM`?

Reply:
Was used before, removed export